src/status_im2/contexts/syncing/view.cljs

```diff
@@ -4,51 +4,70 @@
-    [rn/view {:style styles/device-column}
-     [quo/text
-      {:accessibility-label :device-name
-       :weight              :medium
-       :size                :paragraph-1
-       :style               {:color colors/neutral-100}} device-name]
+  [{:keys [device
+           this-device?
+           type]}]
+  [rn/view {:style style/device-container}
+   [rn/view {:style style/icon-container}
+    [quo/icon
+     (if (= type :mobile) :i/mobile :i/desktop)
 
-     [quo/text
-      {:accessibility-label :device-status
-       :weight              :regular
-       :size                :paragraph-2
-       :style               {:color colors/neutral-50}} device-status]]]])
+     {:color colors/white}]]
+   [rn/view {:style style/device-details}
+    [quo/text
+     {:accessibility-label :device-name
+      :weight              :medium
+      :size                :paragraph-1
+      :style               {:color colors/white}} device]
+    [quo/text
+     {:accessibility-label :next-back-up
+      :size                :paragraph-2
+      :style               {:color colors/white-opa-40}} "Next backup in 04:36:12"]
+    (when this-device?
+      [rn/view {:margin-top 8}
+       [quo/status-tag
+        {:size           :small
+         :status         {:type :positive}
+         :no-icon?       true
+         :label          (i18n/label :t/this-device)
+         :override-theme :dark}]])]])
 
 (defn views
   []
-  [rn/view {:style styles/container-main}
-   [quo/text
-    {:accessibility-label :synced-devices-title
-     :weight              :medium
-     :size                :paragraph-2
-     :style               {:color colors/neutral-50}} (i18n/label :t/synced-devices)]
-   [rn/view {:style styles/devices-container}
-    [render-device "iPhone 11" (i18n/label :t/this-device)] ;; note : the device name is hardcoded for
-                                                            ;; now
-    [rn/view {:style styles/sync-device-container}
+  [rn/view {:style style/container-main}
+   [navigation-bar]
+   [rn/view {:style style/page-container}
+    [rn/view {:style style/title-container}
+     [quo/text
+      {:size   :heading-1
+       :weight :semi-bold
+       :style  {:color colors/white}} (i18n/label :t/syncing)]
      [quo/button
-      {:label    :primary
-       :size     40
-       :before   :i/placeholder
+      {:size     32
+       :icon     true
        :on-press (fn []
                    (rf/dispatch [:bottom-sheet/hide])
                    (rf/dispatch [:bottom-sheet/show-sheet
                                  {:show-handle? false
                                   :content      (fn []
-                                                  [sync-device-notice/sheet])}]))}
-      (i18n/label :t/sync-new-device)]]]])
+                                                  [sync-device-notice/sheet])}]))} :i/add]]
+    [rn/view {:style style/devices-container}
+     [render-device
+      {:device       "iPhone 11"
```

Comment:
Quick question: Am I correct in assuming we'll eventually check the device model to decide which name to display? Just so we don't show the text `iPhone 11` to an user of `iPhone 14`, and so on.

Reply:
yes definitely! I have a TODO at the top of the file to adjust this mock data as part of the larger story